pkg/daprrp/setup/setup.go

```diff
@@ -0,0 +1,149 @@
+/*
+Copyright 2023 The Radius Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package setup
+
+import (
+	"time"
+
+	asyncctrl "github.com/radius-project/radius/pkg/armrpc/asyncoperation/controller"
+	"github.com/radius-project/radius/pkg/armrpc/builder"
+	apictrl "github.com/radius-project/radius/pkg/armrpc/frontend/controller"
+	"github.com/radius-project/radius/pkg/daprrp/datamodel"
+	"github.com/radius-project/radius/pkg/daprrp/datamodel/converter"
+	"github.com/radius-project/radius/pkg/recipes/controllerconfig"
+
+	ds_ctrl "github.com/radius-project/radius/pkg/daprrp/frontend/controller"
```

Comment:
nit: can we rename this to something like `dapr_ctrl` or similar? `ds_ctrl` is used for datastores so might be confusing